### PR TITLE
591 avoid unnecessary Stopwatch::start_new

### LINKED ISSUE #591: Avoid Stopwatch start_new if log level is higher than trace
This is a micro optimization.
There are several places, where Stopwatch is used and the time measurement goes only to logs on trace level.
Examples:
https://github.com/qoollo/bob/blob/master/bob/src/server.rs#L112
https://github.com/qoollo/bob/blob/master/bob/src/grinder.rs#L82
https://github.com/qoollo/bob/blob/master/bob-backend/src/core.rs#L242

In this places we can avoid Stopwatch start if log level is higher than trace

### PR DESCRIPTION
**Pull request by lostl1ght**
Closes #591

### CODE REVIEW

**Review by idruzhitskiy — APPROVED** [review]

**Comment by ikopylov** [comment]
I think we don't really need this micro-optimization. Please revert everything except changes where `sw.elapsed()` was evaluated outside of the logging functions. And update the description in the CHANGELOG accordingly

File: bob/src/server.rs [hunk 1]
```diff
@@ -200,8 +203,7 @@ where
                 "grinder finished request processing /{:.3}ms/",
                 sw.elapsed().as_secs_f64() * 1000.0
             );
-            let elapsed = sw.elapsed_ms();
-            debug!("GET[{}]-OK dt: {}ms", key, elapsed);
+            debug!("GET[{}]-OK dt: {}ms", key, sw.elapsed_ms());
```
Comment (ikopylov) [inline bob/src/server.rs:206]: Keep this change

File: bob/src/server.rs [hunk 2]
```diff
@@ -236,8 +241,7 @@ where
             .exist(&keys, &options)
             .await
             .map_err::<Status, _>(|e| e.into())?;
-        let elapsed = sw.elapsed();
-        debug!("EXISTS-OK dt: {:?}", elapsed);
+        debug!("EXISTS-OK dt: {:?}", sw.elapsed());
```
Comment (ikopylov) [inline bob/src/server.rs:244]: And this one too

**Comment by ikopylov** [comment]
This optimization is not significant

**Pull request closed without merging**